.github/workflows/nightly.yml

```diff
@@ -105,7 +105,7 @@ jobs:
         uses: actions/setup-java@v2
         with:
           java-version: ${{ env.JDK_VERSION }}
-          distribution: 'adopt'
+          distribution: 'zulu'
```

Comment:
Can you plase make distribution name a variable as `JDK_VERSION`?

Reply:
Yes. Actually I think you mean JDK_DISTRO. IMO, I prefer putting values in a matrix (array) than using env.JDK_DISTRO variable.
Such as:
```
strategy:
      matrix:
        os: [ ubuntu-latest ]
        java-version: [ 16 ]
        distro: [ 'zulu', 'temurin' ]
        
        
        // .. rest
        distribution: ${{ matrix.distro }}
        
```
Is this what you mean?

Reply:
Yes, that's what I was thinking of. Thanks

Reply:
For build I will add a matrix for os, java-version, distro for both files. The build.yml will have both zulu and temurin. the nightly will just have zulu. Because they are both great distros however, have dmg, for two different distros might be duplicate effort. Both are TCK tested builds, but Temurin may not support most non-LTS releases. For example JDK 18 if you plan to play with newer features.

Reply:
Please do not matrix for builders.